curp/src/server/curp_node.rs

```diff
@@ -226,6 +228,10 @@ impl<C: 'static + Command> CurpNode<C> {
             };
 
             let result = async {
+                if batch_timeout_flag && ae.entries.is_empty() {
+                    batch_timeout_flag = false;
+                    return Ok(());
+                }
```

Comment:
I guess you don't need another flag here. Just use `hb_opt` flag here.

Reply:
Fixed